### ISSUE TITLE
Breaks in Hyprland v0.36.0

### ISSUE THREAD

**Issue by donovanglover** [donovanglover/hyprdim#78]
Just a heads up that hyprdim breaks with Hyprland v0.36.0. Likely depends on [hyprland-rs](https://github.com/hyprland-community/hyprland-rs) being updated since the issue is serde related.

```
Error: SerdeError(Error("missing field `int`", line: 1, column: 71))
```

## Priorities
Add a :+1: [reaction] to [issues you find important].

[reaction]: https://github.blog/2016-03-10-add-reactions-to-pull-requests-issues-and-comments/
[issues you find important]: https://github.com/donovanglover/hyprdim/issues?q=is%3Aissue+is%3Aopen+sort%3Areactions-%2B1-desc

**Comment by donovanglover** [comment]
Upstream issue: https://github.com/hyprland-community/hyprland-rs/issues/184

**Comment by donovanglover** [comment]
Fixed in v2.2.4
